node/docs/releases.md

```diff
@@ -1,5 +1,8 @@
 # VSTS-TASK-LIB RELEASES
 
+## 2.0.1-preview
```

Comment:
i have another branch in parallel for other 2.0.1 changes, but wanted to discuss this particular set of changes separately